v2/pom.xml

```diff
@@ -680,19 +680,19 @@
         <module>file-format-conversion</module>
         <module>googlecloud-to-elasticsearch</module>
         <module>googlecloud-to-googlecloud</module>
+        <module>googlecloud-to-mongodb</module>
+        <module>hive-to-bigquery</module>
```

Comment:
I think hive-to-bigquery module needs to be removed here, otherwise the project won't build. It was removed from the project (and I don't see the module itself in your branch either).

Reply:
Removed.